### PR TITLE
Option to Remove Psssword

### PR DESCRIPTION
**Pull request by shiv07tiwari**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [ ] `./gradlew checkstyle`

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java [hunk 1]
```diff
@@ -142,6 +142,9 @@ public void onSelection(MaterialDialog dialog, View view, int which, CharSequenc
                                     case 6://Password Set
                                         mPDFUtils.setPassword(filePath, ViewFilesAdapter.this);
                                         break;
+                                    case 7://Password  Remove
+                                        mPDFUtils.removePassword(filePath, ViewFilesAdapter.this,mFileList);
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java]: add `,` after this

File: app/src/main/java/swati4star/createpdf/util/PDFUtils.java [hunk 2]
```diff
@@ -143,5 +141,105 @@ public void onClick(DialogInterface dialogInterface, int i) {
...
+                } catch (IOException e) {
+                    e.printStackTrace();
+                }
+                byte[] password;
+                finalOutputFile = file.replace(".pdf", mContext.getString(R.string.decrypted_file));
+                for (int i = 0; i < FileList.size(); i++) {
+
+                    if(finalOutputFile.equals(FileList.get(i).getPath())) {
+                        int append = FileUtils.checkRepeat(finalOutputFile, FileList);
+                        finalOutputFile = finalOutputFile.replace(".pdf",append+".pdf");
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/PDFUtils.java]: add space after ,

Comment (Swati4star) [reply app/src/main/java/swati4star/createpdf/util/PDFUtils.java]: add space before & after +

**Comment by Swati4star** [comment]
@shiv07tiwari Fix the checkstyle errors.

**Comment by Swati4star** [comment]
@shiv07tiwari still failing...

**Comment by shiv07tiwari** [comment]
@Swati4star  Fixed it

**Pull request merged**